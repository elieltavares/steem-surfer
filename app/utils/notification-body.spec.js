/* eslint-disable */

import notificationBody from './notification-body';

describe('Notification Body', () => {
  it('(1) Vote', () => {
    const input = {
      type: 'vote',
      source: 'foo'
    };

    expect(notificationBody(input)).toMatchSnapshot();
  });

  it('(2) Mention - post', () => {
    const input = {
      type: 'mention',
      source: 'bar',
      extra: {
        is_post: true
      }
    };

    expect(notificationBody(input)).toMatchSnapshot();
  });

  it('(3) Mention - comment', () => {
    const input = {
      type: 'mention',
      source: 'bar',
      extra: {

      }
    };

    expect(notificationBody(input)).toMatchSnapshot();
  });

  it('(4) Follow', () => {
    const input = {
      type: 'follow',
      source: 'bar'
    };

    expect(notificationBody(input)).toMatchSnapshot();
  });

  it('(5) Reply', () => {
    const input = {
      type: 'reply',
      source: 'bar'
    };

    expect(notificationBody(input)).toMatchSnapshot();
  });

  it('(6) Reblog', () => {
    const input = {
      type: 'reblog',
      source: 'bar'
    };

    expect(notificationBody(input)).toMatchSnapshot();
  });

});
