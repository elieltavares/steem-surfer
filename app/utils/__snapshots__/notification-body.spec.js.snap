// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification Body (1) Vote 1`] = `"foo voted your post"`;

exports[`Notification Body (2) Mention - post 1`] = `"bar mentioned you in a post"`;

exports[`Notification Body (3) Mention - comment 1`] = `"bar mentioned you in a comment"`;

exports[`Notification Body (4) Follow 1`] = `"bar followed you"`;

exports[`Notification Body (5) Reply 1`] = `"bar replied you"`;

exports[`Notification Body (6) Reblog 1`] = `"bar reblogged your post"`;
